components/navigation.css

```diff
@@ -0,0 +1,52 @@
+.nav-container {
+    position: relative;
+    max-width: 900px;
+    height: 300px;
```

Comment:
Use rem in place of px for responsiveness.

Reply:
Oh thank you for pointing it out it was there by mistake.